libvmaf/src/feature/third_party/funque/integer_funque_filters.c

```diff
@@ -302,18 +319,27 @@ static inline void integer_horizontal_filter(spat_fil_inter_dtype *tmp, spat_fil
 
 }
 
-void integer_spatial_filter(void *src, spat_fil_output_dtype *dst, int width, int height, int bitdepth)
-{
+const spat_fil_coeff_dtype i_ngan_filter_coeffs[21] = {
+    -900,  -1054, -1239, -1452, -1669, -1798, -1547, -66,   4677,  14498, 21495,
+    14498, 4677,  -66,   -1547, -1798, -1669, -1452, -1239, -1054, -900};
 
-    const spat_fil_coeff_dtype i_filter_coeffs[21] = {
-        -900, -1054, -1239, -1452, -1669, -1798, -1547, -66, 4677, 14498, 21495,
-        14498, 4677, -66, -1547, -1798, -1669, -1452, -1239, -1054, -900
-    };
+const spat_fil_coeff_dtype i_nadeanu_filter_coeffs[5] = {1658, 15139, 31193, 15139, 1658};
 
-    int src_px_stride = width;
-    int dst_px_stride = width;
+void integer_spatial_filter(void *src, spat_fil_output_dtype *dst, int dst_stride, int width,
+                            int height, int bitdepth, spat_fil_inter_dtype *tmp, int num_taps)
+{
+    int fwidth;
```

Comment:
`fwidth` appears to be a duplicate of `num_taps`.

Reply:
We will address num_taps redundancy in the next commit